content/master/concepts/managed-resources.md

```diff
@@ -449,6 +471,35 @@ Note that if a resource has required fields, you must fill those fields or the
 creation of the managed resource will be rejected. So, in those cases, you will
 need to enter the name of the resource as well as the required fields.
 
+### Alternative Import Procedure: Start with ObserveOnly
+
+The above approach has the following caveats:
+
+1. You need to provide all the required fields in the spec of the resource with
+correct values even though they are not used for importing the resource. A wrong
+value for a required field will result a configuration update which is not
+desired.
+2. Any typo in the external name annotation or some mistake in the identifying
+arguments (e.g. `region`) will result creation of a new resource instead of
+importing the existing one.
```

Comment:
```suggestion
2. Any typos in the external name annotation or mistakes in the identifying arguments, such as the region, results in the creation of a new resource instead of importing the existing one.
```